app/helpers/benchmarks_helper.rb

```diff
@@ -0,0 +1,63 @@
+module BenchmarksHelper
+  IDSEQ_BENCH_BUCKET = "idseq-bench".freeze
+  IDSEQ_BENCH_KEY = "config.json".freeze
+
+  def benchmarks_list
+    benchmarks = {}
+
+    # Read benchmarks from configuration store in S3
+    benchmark_config = benchmarks_config()
+    if benchmark_config
+      benchmarks = parse_config(benchmark_config)
+
+      benchmarks[:active_benchmarks].each do |benchmark|
+        # get the most recent finished sample of this benchmark
+        sample = Sample
+                 .includes(:pipeline_runs)
+                 .where(project: Project.find_by(name: benchmark[:project_name]))
+                 .last
+        benchmark[:last_run] = nil
+        if sample
+          benchmark[:last_run] = {
+            sample_name: sample.name,
+            pipeline_version: sample.first_pipeline_run.version
+          }
+        end
+      end
+    end
+
+    return benchmarks
+  end
+
+  def benchmarks_config
+    s3_response = S3_CLIENT.get_object(bucket: IDSEQ_BENCH_BUCKET, key: IDSEQ_BENCH_KEY)
+    return JSON.parse(s3_response.body.read, symbolize_names: true)
+  rescue Aws::S3::Errors::NoSuchKey
+    LogUtil.log_err_and_airbrake("[BenchmarkController] Config file not found - '#{IDSEQ_BENCH_KEY}'")
```

Comment:
nit: actually BenchmarksController

Reply:
This was actually in the helper now! Removed the prefix.